tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -76,29 +76,29 @@ public async Task ARLINGTON_ROPC_AAD_Async()
         public async Task ARLINGTON_ROPC_ADFS_Async()
         {
             var labResponse = await LabUserHelper.GetArlingtonADFSUserAsync().ConfigureAwait(false);
-            await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
+            await RunHappyPathTestAsync(labResponse, isAad: true).ConfigureAwait(false);
```

Comment:
when i run the adfs tests, we use an assertion, so we don't have a username to send, so i'm not sure how the adfs tests worked previously.

Reply:
Not sure what you mean. The ropc requires the user to pass in the username so we simply take that value and construct the header with it. Even for adfs accounts

![image](https://user-images.githubusercontent.com/30090357/122450926-71e97c00-cf5c-11eb-9ceb-6c4843cb0ccd.png)

Reply:
this is what i see when i tested:
![image](https://user-images.githubusercontent.com/19942418/122459181-76fef900-cf65-11eb-8ccf-8fd4891dcbe6.png)

Reply:
Ok, I think I see what is happening here. We used to construct the CCS header in the request which allows for the use of the _UserNamePasswordParameters which is available at the request level. Since this PR has it in the tokenClient. those values are not available.

I remember running into this and had to make this tweak:

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/d8e77fe072d4326027ed7f75c97179a03ba0897e/src/client/Microsoft.Identity.Client/Internal/Requests/UsernamePasswordRequest.cs#L136-L139

Reply:
ccs is not supported with ADFS correct? so we don't even need it for ADFS.

Reply:
Right.

Reply:
However, Im not 100% sure about hybrid environments. Ill reach out to the CCS contact

Reply:
sounds good. thanks.

Reply:
Just got word from the CCS guys. ADFS is supported with CCS

Reply:
interesting, when did they start supporting it? i asked last week, and unless i understood the [discussion wrong](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2687#discussion_r649578911), i thought it was not supported with ADFS.

Reply:
I think that may have been a misunderstanding on our part. There is an email from the CCS team just now that says it is supported.